frontend/src/components/ccofApplication/group/OrganizationInformation.vue

```diff
@@ -61,7 +61,7 @@
                     :disabled="isLocked"
                     variant="outlined"
                     required
-                    :rules="rules.required"
+                    :rules="[...rules.required, rules.maxLength(MAX_CHAR_LENGTH)]"
```

Comment:
As discussed, let's remove this validation. I'm just cautious of implications for existing data.

Reply:
they gone